### DISCUSSION TITLE
building SofaCaribou with Sofa v22.06.00 on MacOS

### DISCUSSION THREAD

**Discussion by veruozzi** [jnbrunet/caribou#139] [General]
Hello @jnbrunet,

I'm trying to build caribou plugin but I still get errors in cmake configuration step. 
I'm using the version v22.06 of SofaFramework and in particular I've downloaded the MacOS binaries from the website -- in this version is already installed SofaPython3 so I've set my env variables in this way:

$ export SOFA_ROOT=my_path/SofaInstall
$ export SP3_ROOT=$SOFA_ROOT/plugins/SofaPyhton3

I've cloned caribou in an external repo and in the same repo I've created a CMakeLists file:

> cmake_minimum_required(VERSION 3.12)
> 
> 
> find_package(SofaFramework)
> 
> 
> sofa_add_subdirectory(plugin caribou/  Caribou VERSION 21.06.00)

On CMake I've set as CMAKE_PREFIX_PATH the path to sofa install.
nevertheless I get this error:

> CMake Error at /Applications/SOFA_v22.06.00_MacOS/collections/SofaGui/lib/cmake/SofaGui/SofaGuiTargets.cmake:91 (message):
  The imported target "runSofa" references the file

     "/Applications/SOFA_v22.06.00_MacOS/collections/SofaGui/bin/runSofa-22.06.00"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "/Applications/SOFA_v22.06.00_MacOS/collections/SofaGui/lib/cmake/SofaGui/SofaGuiTargets.cmake"

  but not all the files it references.

Do I make some mistakes in the building set-up? Or in the env variables?

**Comment by jnbrunet** [comment]
Hey @veruozzi , it looks like it is coming from SOFA's side and not Caribou. I just created an issue on SOFA github, you can follow the development there: https://github.com/sofa-framework/sofa/issues/3485

Thanks for the report !

**Comment by veruozzi** [comment]
Hi @jnbrunet ,
thank you very much for the creation of the new issue :)
